### PR TITLE
create recipes/rtopy/

### PR DESCRIPTION
**Pull request by thierrymoudiki**
Checklist
- [ x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [ x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x ] Source is from official source.
- [ x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [ x] If static libraries are linked in, the license of the static library is packaged.
- [ x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [ x] Build number is 0.
- [ x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [ x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by thierrymoudiki** [comment]
@conda-forge/help-python, ready for review!

File: LICENSE [hunk 1]
```diff
@@ -1,9 +1,32 @@
-testgres is released under the PostgreSQL License, a liberal Open Source license, similar to the BSD or MIT licenses.
```
Comment (ocefpaf) [inline LICENSE:1]: You should not change this file.

**Review by thierrymoudiki — COMMENTED** [review]
The file apparently caused an issue, and was asked to resolve it. What should I fix?

File: LICENSE (hunk 1 above)
Comment (ocefpaf) [reply LICENSE:1]: Actually, this file seems to be wrong and should be removed. All licenses for packages must be in the package recipe directory.

Comment (thierrymoudiki) [reply LICENSE:1]: I've just deleted the file

File: LICENSE.txt [hunk 2]
```diff
@@ -1,27 +0,0 @@
-Copyright (c) 2015-2023, conda-forge
```
Comment (ocefpaf) [inline LICENSE.txt:1]: This is the correct one and should stay.

Comment (thierrymoudiki) [reply LICENSE.txt]: Done

File: LICENSE (hunk 1 above)
Comment (thierrymoudiki) [reply LICENSE:1]: @conda-forge/help-python, please review

File: recipes/rtopy/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+source:
+  url: https://github.com/Techtonique/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: bd5d3d4a102863764d085b493688c9ad3a138cf0814abf12638c2af81de54935
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+  skip: true  # [py < 35]
+  skip: true  # [win]
```
Comment (ocefpaf) [inline recipes/rtopy/meta.yaml:16]:
```suggestion
  skip: true  # [py<35]
  skip: true  # [win]
```

**Comment by thierrymoudiki** [comment]
👍

**Pull request merged**